### DISCUSSION TITLE
Subler

### DISCUSSION THREAD

**Discussion by gotzlotz** [elrumo/macOS_Big_Sur_icons_replacements#1167] [Icon request]
Anyone have a Subler or can make one for Big Sur. Not really a fan of the stock one.
Thanks

[Subler](https://subler.org/)

**Comment by rodgerwerner** [comment]
Here's one I redid based on Barry Mieny's from awhile back:

![Subler_2](https://user-images.githubusercontent.com/23433246/117758677-78176a80-b1f0-11eb-8a22-fa4dbcdd102d.png)

**Reply by gotzlotz** [reply]
I have been looking all over for this icon.  Thanks 🙏🏾
